Description: Fix cinnamon-settings not working cause by FHS fixes
Author: Fabio Fantoni <fantonifabio@tiscali.it>
Origin: Debian
Forwarded: no (only when FHS fixes will be accepted)
Applied-Upstream: no
--- a/files/usr/lib/cinnamon-settings/cinnamon-settings.py
+++ b/files/usr/lib/cinnamon-settings/cinnamon-settings.py
@@ -29,7 +29,7 @@
     print "No settings modules found!!"
     sys.exit(1)
 
-mod_files = [x.split('/')[5].split('.')[0] for x in mod_files]
+mod_files = [x.split('/')[6].split('.')[0] for x in mod_files]
 
 for mod_file in mod_files:
     if mod_file[0:3] != "cs_":
